book/SUMMARY.md

```diff
@@ -180,37 +213,3 @@
   - [EPG Ingestion](/applicaster-video-platform/content-ingestion/epg-ingestion/index.md)
   - [Collection Ingestion](/applicaster-video-platform/content-ingestion/collection-ingestion-xml/collection_via_xml.md)
   - [Authorization Providers](/applicaster-video-platform/authorization-provider/authorization_provider.md)
-
-# QuickBrick
-
-## QuickBrick
-
-- [Introduction](/quick-brick/intro.md)
-- [Apple TV](/quick-brick/tvOS/intro.md)
-  - [Plugins](/quick-brick/tvOS/Plugins/intro.md)
-    - [Analytics](/quick-brick/tvOS/Plugins/Analytics/intro.md)
-      - [Plugin creation guide](/quick-brick/tvOS/Plugins/Analytics/GeneralAnalyticsHowCreate.md)
-      - [Google Analytics](/quick-brick/tvOS/Plugins/Analytics/GoogleAnalytics/GoogleAnalytics.md)
-    - [Player](quick-brick/tvOS/Plugins/Video/intro.md)
-      - [Plugin creation guide](quick-brick/tvOS/Plugins/Video/VideoPluginsHowCreate.md)
-      - [Default Player](quick-brick/tvOS/Plugins/Video/DefaultPlayer/DefaultPlayer.md)
-      - [Player Dependent](quick-brick/tvOS/Plugins/PlayerDependents/intro.md)
-        - [Plugin creation guide](quick-brick/tvOS/Plugins/PlayerDependents/DependentPluginsHowCreate.md)
-        - [GoogleIMA](quick-brick/tvOS/Plugins/PlayerDependents/VideoAds/GoogleIMA/GoogleIMA.md)
-  - [Environment](/quick-brick/tvOS/Environment/intro.md)
-  - [Apple TV Debug](/quick-brick/tvOS/Environment/DebugBuildIntegration.md)
-  - [Apple TV Release](quick-brick/tvOS/Environment/ReleaseBuildIntegration.md)
-- [Beta Guidelines & information](/quick-brick/Beta-guidelines.md)
-- [Setup your project](/quick-brick/plugin-setup.md)
```

Comment:
This is the latest plugin guide I have written, it's updated and should stay.

Reply:
It stays. It is also linked to from `Getting Started with Zapp Plugins` > `Get Ready to Work` > `Quick-Brick` so gitbook showed that one selected when I clicked the menu item under QB.

I wasn't sure if it's better to keep the one in the `Getting Started with Zapp Plugins` section or the one in the QB section

Reply:
This is a very basic doc to get started with the plugins setup. 
You can add extra information inside the description chapter about where we manage the q.b qiki.
For example: for more information check our QuickBrick docs. (add a link to the main quick-brick page on the left nav-bar)

Reply:
Please check that the docs assets are still available